test/bridge.test.ts

```diff
@@ -21,7 +23,7 @@ import {
 } from "hardhat/types";
 import { solidity } from "ethereum-waffle";
 import config from "../hardhat.config";
-
+import "./wadraymath";
```

Comment:
list what you want to import instead

Reply:
I'm not exporting anything from `./wadraymath` i'm only extending a module, that's why I need to import the ts file